web-test-runner.config.mjs

```diff
@@ -39,4 +40,18 @@ export default /** @type {import("@web/test-runner").TestRunnerConfig} */ ({
   // ],
 
   // See documentation for all available options
+    plugins:[
+        importMapsPlugin({
+      inject: {
+        importMap: {
+          imports: {
+            // mock a dependency
+            'msw': '/src/mocks/msw.js',
```

Comment:
`@web/test-runner` import maps is not essential for MSW integration but convenient to keep the `msw` import by StoryBook, unit tests, and MSW handlers code in same manner.